text.txt

```diff
@@ -0,0 +1 @@
+Hello word
```

Comment:
couldn't afford the third L?

Reply:
I being paid per character

Reply:
but that file should be deleted after the test